### PR TITLE
Adding a basic /embed page

### PR DESCRIPTION
**Pull request by sfshaza2**
Do we consider embedded to be fully released and just an advanced topic?

### CODE REVIEW

File: src/embedded.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Embedded support for Flutter
```
Comment (mit-mit) [inline src/embedded.md:2]: There are two potentially separate topics here:

1) Using Flutter on a "embedded device", typically a low-powered hardware device such as a smart-display, a thermostat, or similar

2) Embedding Flutter into a new operating system or environment, for example a new mobile platform or a new operating system

These two things are connected, but not necessarily the same, right?

Comment (sfshaza2) [reply src/embedded.md:2]: Correct. I can list both out but is there anything else we want to say about this?

Comment (csells) [reply src/embedded.md:2]: I think listing both of those scenarios is sufficient for now.

File: src/embedded.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+description: "Details of how Flutter supports the creation of embedded experiences."
+---
+
+If you would like to embed Flutter engine into a car,
+a refrigerator, a thermostat... you CAN! 
+However, this feature, while stable,
+uses low-level API and is _not_ for beginners. 
+In addition to the following resources, you
+might consider joining Discord, where Flutter
```
Comment (domesticmouse) [inline src/embedded.md:11]: This feels like a CTA without a link target

**Review by domesticmouse — APPROVED** [review]

**Review by zanderso — APPROVED** [review]

**Pull request merged**